api-lib/event_triggers/createVouchedUser.ts

```diff
@@ -50,7 +50,7 @@ async function handler(req: VercelRequest, res: VercelResponse) {
       const { insert_users_one } = await adminClient.mutate(
         {
           insert_users_one: [
-            { object: { name, address, circle_id } },
+            { object: { name, address, circle_id, entrance: 'vouched in' } },
```

Comment:
Let's make all the `entrance` values have no spaces, so `vouched-in`, `admin-self-add` etc, so its a bit easier in logs and dbs to search and parse these.